suite-native/device-manager/src/components/DeviceItem.tsx

```diff
@@ -24,44 +23,33 @@ const deviceItemWrapperStyle = prepareNativeStyle(utils => ({
     justifyContent: 'space-between',
     alignItems: 'center',
     borderRadius: utils.borders.radii.medium,
-    backgroundColor: utils.colors.backgroundTertiaryDefaultOnElevation0,
+    backgroundColor: utils.colors.backgroundTertiaryDefaultOnElevation1,
     paddingHorizontal: utils.spacings.medium,
     paddingVertical: 12,
 }));
 
 export const DeviceItem = ({ id }: DeviceItemProps) => {
     const dispatch = useDispatch();
+    const { applyStyle } = useNativeStyles();
 
-    const deviceLabel = useSelector((state: DeviceRootState) => selectDeviceLabel(state, id));
-    const deviceName = useSelector((state: DeviceRootState) => selectDeviceName(state, id));
     const device = useSelector((state: DeviceRootState) => selectDeviceById(state, id));
     const currentDevice = useSelector(selectDevice);
 
-    const { applyStyle } = useNativeStyles();
-
     const { setIsDeviceManagerVisible } = useDeviceManager();
 
-    if (!deviceLabel) return null;
-
     const handleSelectDevice = () => {
         setIsDeviceManagerVisible(false);
 
-        if (device?.id === currentDevice?.id) return;
+        if (id === currentDevice?.id) return;
```

Comment:
Can you name `id` little bit more specific? I kind of confused here `id` vs `currentDevice.id` vs `device.id`. You don't need to change prop name just override it somehow like `{ id: deviceItemId }` or smthing like that

Reply:
naming improved as `deviceItemId` and `selectedDeviceId` [4092740](https://github.com/trezor/trezor-suite/pull/10020/commits/4092740700d31090573c5fbf379279c641a91726)